homeassistant/components/overseerr/services.py

```diff
@@ -99,14 +143,164 @@ async def _async_get_requests(call: ServiceCall) -> ServiceResponse:
     return {"requests": cast(list[JsonValueType], result)}
 
 
+async def __search_media(
```

Comment:
```suggestion
async def _search_media(
```
Generally, double underscore prefixes only make sense for methods within classes that are part of inheritance hierarchies where you want to prevent accidental overriding by subclasses. For your case with module-level helper functions, there's no inheritance involved, so a single underscore is the appropriate convention.

Reply:
Okay, I was unsure because of the other method `async_request_media`.

Done in https://github.com/home-assistant/core/pull/155522/commits/28ce93f81e54c3680e3a34fc92c6d1d3d729eb7e